composer.json

```diff
@@ -68,6 +69,7 @@
         }
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "discard-changes": true
```

Comment:
Why is this necessary?

Reply:
Exist issue on drone side (on composer install).
For not completely reasons we have on lower dependencies patched already drupal/coder module and we have error like that:

> RuntimeException]                                                                  
> Source directory /test/oe_translation/vendor/drupal/coder has uncommitted changes.  

By these settings, we enforce composer discard changes and discard changes (patched files)